aries-site/src/examples/components/rangeinput/RangeInputDescriptionExample.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { Box, FormField, RangeInput, Text } from 'grommet';
+
+export const RangeInputDescriptionExample = () => {
+  const [value, setValue] = useState(10);
+
+  return (
+    <FormField label="Label" help="RangeInput Description">
+      <Box
+        pad={{ horizontal: '11px', vertical: '5px' }}
```

Comment:
Why the custom pixel value padding?

Reply:
so with the `inputs` we have this in the theme.. [here](https://github.com/grommet/grommet-theme-hpe/blob/master/src/js/index.js#L166)
however, this is not applying to `rangeinput` so that is why I have it here within the `box` wrapping it.. I agree this is not ideal so I can look at how we can enhance grommet to also have some pad on `rangeinput` otherwise when wrapping this in `formfield` it looks horrible
<img width="904" alt="Screen Shot 2021-09-22 at 9 41 03 AM" src="https://user-images.githubusercontent.com/42451602/134376113-a7b5e77d-c04d-4b6f-8587-32fe630123a6.png">

Reply:
We should definitely address this in grommet/theme.

Reply:
I agree Ill peak around today!